### DISCUSSION TITLE
Timeline for the next release

### DISCUSSION THREAD

**Discussion by moonflare** [mirumee/ariadne#1062] [Q&A] [answered]
Hello team,

I was wondering if you have a timeline for the next release, so my team and I know how to plan things internally in the meantime. cc @rafalp 🙏🏻  Thanks!

**Comment by rafalp** [comment, marked as answer]
Hello,

Is there anything on changelog you are requiring for next release?

We don't do timelines for releases unless next release is imminent. From how things are currently, Ariadne 0.19 should be released early next week, once documentation for it is completed.

**Comment by moonflare** [comment]
Thanks! We are primarily in need of the `@interfaceObject` support, and early next week sounds great 🙏🏻 
Please let me know if there's any help needed documenting the `@interfaceObject` part 📝
